Src/BootCamp.Chapter/Checks.cs

```diff
@@ -1,39 +1,215 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     // This class is used to have a freedom of design, but with tests applied.
+
+    //TODO All the string concatination from a loop should be done using StringBuilder. Please refactor.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)